integration/solana/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Integration tests with Solang and Solana",
   "scripts": {
     "test": "tsc; ts-node setup.ts; mocha --parallel *.spec.ts",
+    "test-anchor": "tsc; ts-node setup.ts; mocha --parallel anchor/*spec.ts",
```

Comment:
So now `yarn test` (or `npm test`) no longer tests everything. How is that right?

How about three scripts:
test: tests everything, as you would expect
test-call-anchor: test the call anchor test
test-other: test the rest

Reply:
I understood it so that just specific anchor tests were move away from testing everything. But your suggestion about having 3 test targets would be even better

Reply:
Before having Anchor tests, we would run `npm run build` and `npm run test`. The new Anchor tests cannot follow the same order of commands to be run, so I believe having a single command for both Solana and Anchor tests is not a good idea. The Anchor test needs many setup commands before we can run `npm run test-anchor`.

Perhaps, it is a better solution to move the Anchor Typescript test to `integration/anchor`. What do you think @seanyoung and @xermicus ?

Reply:
This might be a good idea.

Reply:
> This might be a good idea.

This should be fixed now.